tensorflow_quantum/python/util.py

```diff
@@ -519,3 +519,95 @@ def exponential(operators, coefficients=None):
                                   ] + w_dagger + u_dagger
             circuit += cirq.Circuit(exp_circuit)
     return circuit
+
+
+# ==============================================================================
+# Unsigned quantum integer functions.
+# ==============================================================================
+
+
+def projector_on_one(qubit):
```

Comment:
Don't think this is complicated enough to where it needs it's own function.

Reply:
Good point, removed this